pkg/zabbix/models.go

```diff
@@ -67,8 +67,8 @@ type ItemTag struct {
 type Trend []TrendPoint
 
 type TrendPoint struct {
-	ItemID   string `json:"itemid,omitempty"`
-	Clock    int64  `json:"clock,omitempty,string"`
+	ItemID   int64 `json:"itemid,omitempty"`
```

Comment:
```
pkg/datasource/response_handler.go:24:24: cannot use point.ItemID (type int64) as type string in map index
pkg/datasource/response_handler.go:25:15: cannot use point.ItemID (type int64) as type string in map index
pkg/datasource/response_handler.go:26:13: cannot use point.ItemID (type int64) as type string in map index
pkg/datasource/response_handler.go:28:27: cannot use point.ItemID (type int64) as type string in map index
```
But this is a good idea, I'll take a look closer.

Reply:
24,26c24,26
< 		pointItem := itemsMap[point.ItemID]
< 		if seriesMap[point.ItemID] == nil {
< 			seriesMap[point.ItemID] = timeseries.NewTimeSeriesData()
---
> 		pointItem := itemsMap[strconv.FormatInt(point.ItemID,10)]
> 		if seriesMap[strconv.FormatInt(point.ItemID,10)] == nil {
> 			seriesMap[strconv.FormatInt(point.ItemID,10)] = timeseries.NewTimeSeriesData()
28c28
< 		pointSeries := seriesMap[point.ItemID]
---
>